src/main/java/core/basesyntax/strategy/OperationProcess.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+
+public interface OperationProcess {
+    void operationProcess(FruitTransaction fruitTransaction);
+}

```

Comment:
let's rename it to OperationHandler and its implementations, for example BalanceOperationHandler